src/Bicep.Cli.IntegrationTests/LintCommandTests.cs

```diff
@@ -154,8 +160,8 @@ public async Task Lint_Invalid_SingleFile_ShouldFail_WithExpectedErrorMessage(Da
     {
         var outputDirectory = dataSet.SaveFilesToTestDirectory(TestContext);
         var bicepFilePath = Path.Combine(outputDirectory, DataSet.TestFileMain);
-        var defaultSettings = CreateDefaultSettings();
-        var diagnostics = await GetAllDiagnostics(bicepFilePath, defaultSettings.ClientFactory, defaultSettings.TemplateSpecRepositoryFactory);
+        var defaultSettings = new InvocationSettings();
```

Comment:
Considering that the pattern appears multiple times, it might be worth introducing a static `InvocationSettings.Default` member.

Reply:
done